### PR TITLE
Add ArtSound integration

### PR DESCRIPTION
**Pull request by dukeofphilberg**
## Proposed change

This PR adds the ArtSound integration as a virtual integration for the LinkPlay integration. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/122636
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5704

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced documentation for the ArtSound integration, allowing users to connect and control ArtSound media players.
	- Enhanced user interaction with media playback devices through LinkPlay technology.
	- Added links to additional resources for further information on the LinkPlay integration.

### CODE REVIEW

File: source/_integrations/artsound.markdown [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: ArtSound
+description: Connect and control your ArtSound media players using the LinkPlay integration
+ha_release: 2024.8
```
Comment (joostlek) [inline source/_integrations/artsound.markdown]:
```suggestion
ha_release: 2024.9
```

File: source/_integrations/artsound.markdown [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - Media player
+ha_domain: artsound
+ha_integration_type: virtual
+works_with:
+  - linkplay
+---
+
+The **ArtSound** {% term integration %} allows users to control their ArtSound media players through the **LinkPlay** {% term integration %}.
+
+[Learn more about LinkPlay in Home Assistant.](/integrations/linkplay/)
```
Comment (c0ffeeca7) [inline source/_integrations/artsound.markdown:15]:
```suggestion
[Learn more about LinkPlay in Home Assistant.](/integrations/linkplay/)

{% include integrations/supported_brand.md %}
```

**Comment by c0ffeeca7** [comment]
I'm [drafting this PR](https://developers.home-assistant.io/docs/review-process#prs-are-being-drafted-when-changes-are-needed), as changes are needed. Please undraft it, once the changes have been implemented and the PR is ready for review again.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @dukeofphilberg :+1:

**Pull request merged**